docs/docs/how-shadowing-works.mdx

```diff
@@ -24,10 +24,7 @@ configures two sibling themes:
 
 ```js:title=gatsby-config.js
 module.exports = {
-  plugins: [
-    "gatsby-theme-tomato-blog",
-    "gatsby-theme-tomato-portfolio",
-  ],
+  plugins: ["gatsby-theme-tomato-blog", "gatsby-theme-tomato-portfolio"],
```

Comment:
why is this the only file that changed? is there something different in the prettier config for the monorepo that's different from this one? Also I think it might have been done like this for clarity.

Reply:
I used the same settings from the main repo and ran Prettier on all files (`npx prettier —write '**/*'`
This should be done in the main repo too I guess.